playbooks/cluster_setup_debian_centos.yaml

```diff
@@ -0,0 +1,71 @@
+# This playbook setup SEAPATH on both debian and CentOS machines
```

Comment:
I preferred naming it `cluster_setup.yaml` or `cluster_setup_main.yaml` and merging this file with cluster_setup_debian.yaml and cluster_setup_centos.yaml.

Reply:
I would keep the OS specific playbooks (as written in another comment),
But this playboooks should be renamed, Would cluster_setup_all.yaml be a good name?

Reply:
Ok